db/seeds.rb

```diff
@@ -20,4 +20,77 @@
                email: email,
                password:              password,
                password_confirmation: password)
-end
\ No newline at end of file
+end
+
+
+# Make a rhizome
+Rhizome.create!(
+  name: "Test rhizome"
+  )
+
+ParticleDevice.create!(
+  device_id: Faker::Lorem.characters(24),
+  encrypted_access_token: "S3jF/ramSAL8dZ4zzASXeuBunWv6tLn9jzcU4ABEflfGQkrwbM44CpWMtyRhkED/",
+  rhizome_id: 1
+  )
+
+
+# Make an equipment profile
+EquipmentProfile.create!(
+  name: "Test EP")
+
+HeatingElement.create!(
+  pins: {"power_pin": "1", "control_pin": "0"},
+  equipment_profile: EquipmentProfile.find(1) )
+
+TemperatureSensor.create!(
+  pins: {"data_pin": "2", "onewire_index": "10"},
+  equipment_profile: EquipmentProfile.find(1))
+
+Thermostat.create!(
+  equipment_profile: EquipmentProfile.find(1),
+  element: HeatingElement.create!(
+    pins: {"power_pin": "3", "control_pin": "5"},
+    equipment_profile: EquipmentProfile.find(1)
+    ),
+  sensor: TemperatureSensor.create!(
+    pins: {"data_pin": "4", "onewire_index": "9"},
+    equipment_profile: EquipmentProfile.find(1)
+    )
+  )
+  
+
+# Make a schedule and tasks
+Schedule.create!(
+  name: "Test Sched 1",
+  root_task_id: 1
+  )
+
+Task.create!(
+  type: "TurnOnTask",
+  status: 0,
+  update_data: {"state": "ON"},
+  equipment_id: 1,
+  schedule_id: 1,
+  duration: 60,
+  )
+
+Task.create!(
+  type: "TurnOnTask",
+  status: 0,
+  update_data: {"state": "ON"},
+  equipment_id: 2,
+  schedule_id: 1,
+  duration: 15,
+  parent_id: 1,
+  trigger: 0
+  )
+
+
+# Make a Recipe
+Recipe.create!(
+  name: "Test Recipe",
+  type: "BeerRecipe",
+  schedule_id: 1,
```

Comment:
You should either use #find to look up an existing Schedule or capture the ID of the Schedule created earlier in this script in a variable and use it here. While the seeds script is probably going to be run on an empty database, programmatically specifying the ID's is a good idea and makes it easier to read what's going on in here.

You should probably double-check the other ID assignments elsewhere while you're at it.